app/src/main/java/com/sopt/dive/ui/components/CommonInputField.kt

```diff
@@ -0,0 +1,63 @@
+package com.sopt.dive.ui.components
+
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun CommonInputField(
+    titleText: String,
+    value: String,
+    onValueChange: (String) -> Unit,
+    placeMessage: String,
+    keyboardOptions: KeyboardOptions,
+    visualTransformation: VisualTransformation? = null,
+    errorMessage: String? = null
+
+    ) {
+
+    Column() {
+
+        Text(titleText.uppercase().toString(), modifier = Modifier.fillMaxWidth())
+
+        TextField(
+            value = value,
+            onValueChange = { onValueChange(it) },
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(color = Color.Transparent)
+                .padding(bottom = 40.dp),
+            colors = TextFieldDefaults.colors(
+                focusedContainerColor = Color.White, unfocusedContainerColor = Color.Transparent
+            ),
+            isError = errorMessage != null,
+            supportingText = {
+                if (errorMessage != null) {
+                    Text(
+                        text = errorMessage, fontSize = 12.sp
+                    )
+                }
+            },
+            placeholder = { Text(placeMessage) },
+            singleLine = true,
+            keyboardOptions = keyboardOptions,
+            visualTransformation = visualTransformation ?: VisualTransformation.None
```

Comment:
visualTransformation의 null 값인 경우에 대한 처리를 여기서 하지않고
visualTransformation를 위에서 선택 파라미터로 만들어 다음과 같이 적어주시는건 어떠실까요??

```suggestion
            visualTransformation = visualTransformation
```

(위에 파라미터)
` visualTransformation: VisualTransformation? = null,
`
-> `visualTransformation: VisualTransformation = VisualTransformation.None,
`

Reply:
아 이렇게 기본 값으로 선언하면 nullable이 되면서 조금 더 안전한 코드가 될 수 있을 것 같네요 ! 감사합니다 !